coinswap-react-native/build-scripts/release/build-release-android-arm64_v8a.sh

```
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ -z "${ANDROID_NDK_ROOT:-}" ]; then
+  if [ -n "${ANDROID_HOME:-}" ] && [ -d "$ANDROID_HOME/ndk" ]; then
+    ANDROID_NDK_ROOT=$(ls -d "$ANDROID_HOME/ndk/"*/ 2>/dev/null | sort -V | tail -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)